animata/card/subscribe-card.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import FeatureCard from "@/animata/card/feature-card";
+import { Meta, StoryObj } from "@storybook/react";
+
+const meta = {
+  title: "Card/Feature Card",
```

Comment:
Change this to `Card/Subscribe Card` as well.

Reply:
@hari i changed all these now in vscode I will push, but http://[localhost:3000/docs/card/subscribe-card](http://localhost:3000/docs/card/subscribe-card) will work right, its already set when created component, how I should change path ..please help